composer.json

```diff
@@ -29,6 +29,7 @@
         "ext-soap": "*",
         "ext-spl": "*",
         "ext-zlib": "*",
+        "composer-runtime-api": "^2",
```

Comment:
should we require (if possible) composer ^2.8 instead instead, just to be sure?

Reply:
I don't think we need to go that high, my composer version is 2.7.9 and was released exactly one month ago. But it looks like the runtime-api version is different anyway:

```sh
$ composer show -p composer-runtime-api
name     : composer-runtime-api
descrip. : The Composer Runtime API
keywords : 
versions : * 2.2.2
type     : library
homepage : 
source   : []  
dist     : []  
names    : composer-runtime-api
```

Reply:
mmm ok, it's just that the more updated everything is the lest weird combination of situations we have to test and support